rapier3d/package.json

```diff
@@ -17,5 +17,8 @@
     },
     "dependencies": {
         "wasm-pack": "^0.12.1"
-    }
+    },
+    "sideEffects": [
+        "./*.js"
+    ]
```

Comment:
- I'm not sure why that's here, that's probably a side effect of building locally since https://github.com/dimforge/rapier.js/pull/289

But I'm not sure why that wasn't committed before.